src/reanimated2/animations.ts

```diff
@@ -179,7 +179,7 @@ export function withStartValue(startValue, animation) {
   });
 }
 
-export function withTiming(toValue, userConfig, callback) {
+export function withTiming(toValue: any, userConfig: any, callback?: any) {
```

Comment:
It is just a temporary solution. We will add types for animation in the next PR.